target_priority.opy

```diff
@@ -0,0 +1,53 @@
+#!mainFile "main.opy"
+
+# Threat. Shows the potential danger of the enemy. This value is only for players, i.e bots always have 0 threat.
+playervar threat
+
+# Attractiveness. Shows how big of a "target" the hero is. For both bots and players (for bots, it can show who to heal, for example)
+playervar attractiveness
+
+# Do note that these values are for each player. It is up to each bot to find their own target, which is a combination of both this value, and some of their own values.
```

Comment:
aren't threat and attractiveness the same thing but just opposite? in the end we are just searching to find who to target, so one formula should be enough

Reply:
yeah i guess so

though there are certain heroes that can have high threat and high attractiveness, such as frontline tanks